mesonbuild/cmake/interpreter.py

```diff
@@ -903,6 +904,19 @@ def configure(self, extra_cmake_options: T.List[str]) -> CMakeExecutor:
         if version_compare(cmake_exe.version(), '>=3.14'):
             self.cmake_api = CMakeAPI.FILE
             self.fileapi.setup_request()
+        else:
+            mlog.warning(f'Support for CMake <3.14 (Meson found {cmake_exe.version()}) is deprecated since Meson 0.61.0')
```

Comment:
Shouldn't we use `mlog.deprecation` here?

Reply:
Yeah, good catch.